core/scripts/flares/activate_mortar.sqf

```diff
@@ -0,0 +1,25 @@
+/*******************************************************************************
+                          Realizado por MIV
+*******************************************************************************/
+
+params ["_mortar"];
+
+_mortar setVehicleAmmo 0.0; 
+{
+	_mortar addmagazine "8Rnd_82mm_Mo_Flare_white"
```

Comment:
not really good indentation here! (and bellow actually)

Reply:
The indentation is set by the VSC. They are two tabs. It's just a matter of how it looks